tests/integration/serialization/.gitignore

```diff
@@ -0,0 +1 @@
+../schema_registry/gen/*
```

Comment:
To avoid requiring protoc to run test-suite I suggest checking in the generated files, what do you think?
Especially since there seems to be some platform-specific assumptions where protoc is installed.